gamedb.yaml

```diff
@@ -4138,3 +4138,16 @@
   name: The Adventure Pals
   compat_tool: proton_513
   
+- name: Unravel Two
+  platform: steam
+  id: 1225570
+  compat_tool: GE-Proton7-28
+  notes:
+    - Requires launcher/installer interaction during first time setup
+    - Uses a launcher which is bypassed after first installation
```

Comment:
Is this not a duplicate of the above line?

Reply:
Hmmm...What I mean is basically:
- First time launching the game, the EA/Origin launcher is installed and requires mouse interaction
- Once the launcher is installed, on next start of the game, the launcher is bypassed and you get straight into the game which can be fuly controlled by controller

Reply:
Yeah, they mean the same thing...